Sources/Orbit/Support/Components/TimelineStep.swift

```diff
@@ -56,73 +71,119 @@ public struct TimelineStep: View {
                 Icon(style.iconSymbol, size: .large, color: style.color)
         }
     }
+}
 
-    var badgeStyle: Badge.Style {
-        switch style {
-            case .default:              return .neutral
-            case .status(let status):   return .status(status, inverted: false)
+// MARK: - Inits
+
+public extension TimelineStep where Header == TimelineStepBadgeText {
+
+    /// Creates Orbit TimelineStep component with custom details.
+    init(
+        _ label: String,
+        sublabel: String,
+        style: TimelineStepStyle,
+        @ViewBuilder contentBuilder: () -> Content
```

Comment:
I would stick with this one to highlight that it's a closure and to make Xcode init suggestion easier to read

Reply:
To distinguish from the other one, maybe lets switch it - the normal one could be called `description` (as our other components do) and the custom one would be `content`, because this is what all other components use and the consistency in the API should be kept.